### PR TITLE
Handle empty command arguments

### PR DESCRIPTION
**Pull request by darkushin**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/47517

## Description
- Fixes an issue where when command arguments were left empty, both `format` and `validate` would fail unexpectedly.
- After the fix, the `format` command will add `[]` to the empty arguments, and the `validate` command will show an indicative error.

### CODE REVIEW

File: demisto_sdk/commands/common/errors.py [hunk 1]
```diff
@@ -224,6 +224,7 @@
     "changed_integration_yml_fields": {'code': "IN147", "ui_applicable": False, 'related_field': 'script'},
     "parameter_is_malformed": {'code': "IN148", 'ui_applicable': False, 'related_field': 'configuration'},
     'empty_outputs_common_paths': {'code': 'IN149', 'ui_applicable': False, 'related_field': 'contextOutput'},
+    "empty_command_arguments": {'code': 'IN150', 'ui_applicable': False, 'related_field': 'arguments'},
```
Comment (darkushin) [inline demisto_sdk/commands/common/errors.py:227]: Not sure whether the `ui_applicable` field should be true or false. Let me know what you think

File: demisto_sdk/commands/common/errors.py [hunk 2]
```diff
@@ -444,6 +445,11 @@ def file_cannot_be_deleted(file_path: str):
     def suggest_fix(file_path: str, *args: Any, cmd: str = 'format') -> str:
         return f'To fix the problem, try running `demisto-sdk {cmd} -i {file_path} {" ".join(args)}`'
 
+    @staticmethod
+    @error_code_decorator
+    def empty_command_arguments(command_name):
+        return f"The arguments of the integration command `{command_name}` are None."
```
Comment (dorschw) [inline demisto_sdk/commands/common/errors.py:451]:
```suggestion
        return f"The arguments of the integration command `{command_name}` must be defined (not None). For an empty list, use `arguments: []`."
```

or whatever the correct syntax for the command is

File: demisto_sdk/commands/common/hook_validations/integration.py [hunk 3]
```diff
@@ -385,6 +385,13 @@ def is_valid_default_argument(self):
 
         for command in commands:
             default_args = set()
+            if command.get('arguments', []) is None:
```
Comment (dorschw) [inline demisto_sdk/commands/common/hook_validations/integration.py:388]: what happens what `arguments` is not in `command`?

**Review by dorschw — COMMENTED** [review]
nice job

File: demisto_sdk/commands/common/hook_validations/integration.py (hunk 3 above)
Comment (darkushin) [reply demisto_sdk/commands/common/hook_validations/integration.py:388]: same as happened before - `command.get('arguments', [])` will return [], which is not None, so the `if` statement will be False and we continue to the old flow which iterates over the arguments list (empty list so it won't really iterate over anything).

**Review by yuvalbenshalom — APPROVED** [review]

**Pull request merged**